startleft/validators/schema.py

```diff
@@ -12,6 +12,7 @@
 class Schema:
     def __init__(self, schema_file_name: str):
         self.schema = self.__load_schema(schema_file_name)
+        logger.debug("OTM schema file loaded successfully")
```

Comment:
Take care. The load_schema method is called for both OTM and mapping definition schemas